index.html

```diff
@@ -410,6 +409,21 @@ <h1 class="contact_form_heading">
     </form>
     </div>
   </footer>
+
+  <div class="mob-menu">
+    <div class="close-tag">
+      <img class="cancel_btn" src="images/Icon_cancel.svg" alt="closing button"/>
```

Comment:
* Kindly make the close icon the same colour as the design template currently its difficult to close the mobile menu without  seeing the icon 
* [ORIGINAL]
![image](https://user-images.githubusercontent.com/52497246/132377510-8b34c057-1537-4443-a0cd-96f3882b9112.png)
* [YOURS]
![image](https://user-images.githubusercontent.com/52497246/132377599-d7a8f4f4-4e1e-49bb-abdb-65aedd1aa308.png)